### DISCUSSION TITLE
Setting up O365

### DISCUSSION THREAD

**Discussion by ptav** [coddingtonbear/django-mailbox#266] [General]
Hi all, thanks to the team for pulling in O365 support!

a note on a detail that wasted me a considerable amount of time: when setting up the URI one must used the App secret VALUE and not the ID.

Would be very useful to the O365 non-initiated if you could highlight this in  the instruction. I would also add that you can safely ignore the Object ID.

thank you

**Comment by Pietro395** [comment]
Hi @ptav ,
You're right, I added the detail regarding the client_secret value in the documentation
